application/app/controllers/dataverse/dataverses_controller.rb

```diff
@@ -24,17 +24,17 @@ def show
       @search_result = @service.search_dataverse_items(params[:id], @page)
       if @dataverse.nil? || @search_result.nil?
         log_error('Dataverse not found.', {dataverse: @dataverse_url, id: params[:id]})
-        flash[:error] = "Dataverse not found. Dataverse: #{@dataverse_url} Id: #{params[:id]}"
+        flash[:alert] = "Dataverse not found. Dataverse: #{@dataverse_url} Id: #{params[:id]}"
```

Comment:
Updated to use the default `alert` and `notice` type from Rails, we can add more to support other types of messages.